lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3662,6 +3684,8 @@ prop_balanceTransactionValid wallet (ShowBuildable partialTx) seed
                         "already balanced"
                     $ classify (txFee tx > Cardano.Lovelace 1_000_000)
                         "fee above 1 ada"
+                    $ classify (hasZeroAdaOutputs $ view #tx partialTx)
+                        "partial tx had zero ada outputs"
```

Comment:
I'm guessing that since we're using `classify`, that we currently don't actually **_require_** that this case actually hits during a given test run?

Is there some way to use `checkCoverage` and `cover` to ensure that we hit this case?

Reply:
I know. Maybe I'll get to it after fixing the `PParam era` vs `W.ProtocolParameters era` and polishing the tests a bit.

For the immediate time, coverage in this property shouldn't matter too much when we have the `computeMinimumCoinForTxOut` properties, a `balanceTx` unit golden, and — soon — integration test coverage through constructTx.